warp10/src/main/java/io/warp10/standalone/StandaloneScriptRunner.java

```diff
@@ -115,11 +122,17 @@ public void run() {
 
           long nano = System.nanoTime();
 
+          HashMap runContext = runContexts.getOrDefault(script, new HashMap());
+          Long execCount = (Long) runContext.getOrDefault(Constants.RUNNER_CONTEXT_EXEC_COUNT, 0L);
+
           WarpScriptStack stack = new MemoryWarpScriptStack(storeClient, directoryClient, props);
-          stack.setAttribute(WarpScriptStack.ATTRIBUTE_NAME, "[StandloneScriptRunner " + script + "]");
+          stack.setAttribute(WarpScriptStack.ATTRIBUTE_NAME, "[StandaloneScriptRunner " + script + "]");
 
           ByteArrayOutputStream baos = new ByteArrayOutputStream();
 
+          long periodicityForNextRun = periodicity;
+          long progForNextRun = Long.MAX_VALUE;
```

Comment:
Rename prod* variables into something clearer. This is not obvious since the name of the function RUNNERPROG was changed.